code/src/components/Thought.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { formatDistanceToNow } from 'date-fns';
+
+const Thought = ({ thought, addLike }) => {
+    return (
+        <div className='thought-container'>
+            <p className="thought-text">{thought.message}</p>
+            <div className="thought-info-container">
+                <button className='btn' onClick={(event) => addLike(thought._id, event)}>
```

Comment:
And on the button maybe:
` onClick={() => addLike(thought._id)}`

I'm not completely sure all this would make the difference but I think it might be a solution :)